src/Trigger/RabbitMQListener.cs

```diff
@@ -221,16 +228,16 @@ private ScaleStatus GetScaleStatusCore(int workerCount, RabbitMQTriggerMetrics[]
                 Vote = ScaleVote.None,
             };
 
-            const int NumberOfSamplesToConsider = 5;
+            int numberOfSamplesToConsider = _sampleCount;
 
-            if (metrics == null || metrics.Length < NumberOfSamplesToConsider)
+            if (metrics == null || metrics.Length < numberOfSamplesToConsider)

```

Comment:
Delete numberOfSamplesToConsider  and use `_sampleCount` directly?

Reply:
Brought back the variables due to the discussion below with Mathew.